torchvision/io/_video_opt.py

```diff
@@ -383,7 +383,7 @@ def get_pts(time_base):
         audio_timebase = info['audio_timebase']
         audio_pts_range = get_pts(audio_timebase)
 
-    return _read_video_from_file(
+    vframes, aframes, info = _read_video_from_file(
```

Comment:
It seems a few arguments of `_read_video_from_file `, such as `video_width`, `video_height`, `video_min_dimension`, are not used here, and can not be specified in API `_read_video`. 

Video frame resizing is fast if done inside of video reader. I would like to keep those argument exposed. 

However, `pyav` backends does not support such resizing. For now, you already raise ValueError when they are not zero. In the long-run, we can consider  applying video resizing transform to keep the same behavior between pyav and video_reader backend.

Reply:
My thinking is that I want to first properly benchmark the benefits of having the rescaling happening in ffmpeg to have an idea of how much improvements it will bring us.

Once we have the numbers, we should analyze them and then decide on how to expose the resizing functionality in a clean way to the users. I think this should be done after December though